fs/operations/operations.go

```diff
@@ -1470,6 +1470,15 @@ func Rcat(fdst fs.Fs, dstFileName string, in io.ReadCloser, modTime time.Time) (
 	return dst, nil
 }
 
+// PublicLink adds a "readable by anyone with link" permission on the given file or folder.
+func PublicLink(f fs.Fs, fileName string) (string, error) {
```

Comment:
This `fileName` should probably be `remote` and be relative to the Fs root as per normal rclone usage.